omod/src/main/java/org/openmrs/module/commonlabtest/web/controller/LabTestOrderPortletController.java

```diff
@@ -67,10 +68,13 @@ protected void populateModel(HttpServletRequest request, Map<String, Object> mod
 						orderJsonArray.add(childJsonObject);
 					}
 				}
+				boolean anyTestRequireSample = false;
+				for (JsonElement element : orderJsonArray) {
+					anyTestRequireSample = ((JsonObject) element).get("requiredSpecimen").getAsBoolean();
+				}
 				System.out.println("Totatl Resullt SET ::" + orderJsonArray);

```

Comment:
Remove console output statements. Rather put them in the log if this information is to be collected.